.taskcluster.yml

```diff
@@ -0,0 +1,315 @@
+---
```

Comment:
This file is just a copy of the one we have in Fenix: https://github.com/mozilla-mobile/fenix/blob/34144fd254f00bd8a4176a296fad384ced69f2bf/.taskcluster.yml

I renamed Fenix references to match firefox-ios.